libs/design/src/molecules/backdrop/backdrop/backdrop.component.scss

```diff
@@ -1,14 +1,16 @@
 @import '../../../scss/daff-util';
+$bg-color: rgba(0, 0, 0, 0.3);
+$highlight-color: rgba(0, 0, 0, 0);
 
 :host {
 	@include clickable();
 	display: block;
-	-webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+	-webkit-tap-highlight-color: $highlight-color;
 	-webkit-tap-highlight-color: transparent;
 }
 
 .daff-backdrop {
-	background: rgba(0, 0, 0, 0.3);
+	background: $bg-color;
```

Comment:
`rgba($black, 0.3);`

Reply:
This is outside the scope of this PR, resolving for sanity.